### DISCUSSION TITLE
MacBook Pro (14-inch, 2021) built-in HDMI not working

### DISCUSSION THREAD

**Discussion by someengineer-nordsec** [MonitorControl/MonitorControl#1020] [Q&A] [answered]
I have two external (identical) displays connected

Display A is connected over https://www.apple.com/shop/product/MUF82AM/A/usb-c-digital-av-multiport-adapter
Display B is connected to built-in HDMI port

I can adjust brightness for A, but cannot for B.
(Personal comment, I would expect the dongle not to work)

Is there anyway that I can attach additional logs?

`Console.app` shows log for `MonitorControl.app` restart

![Screenshot 2022-05-01 at 21 02 46](https://user-images.githubusercontent.com/99187211/166160629-4f4656dc-b6fc-416e-a166-b7fdc12afd0b.png)
![Screenshot 2022-05-01 at 21 02 50](https://user-images.githubusercontent.com/99187211/166160633-18c1d98f-e459-47ea-80be-f23e00bc6e7d.png)
![Screenshot 2022-05-01 at 21 03 31](https://user-images.githubusercontent.com/99187211/166160634-3ea6b045-c8d2-42cb-9204-fca4c2eff8e0.png)
![Screenshot 2022-05-01 at 21 05 01](https://user-images.githubusercontent.com/99187211/166160674-40556848-4c29-4de7-9e17-31d40c7bfa45.png)
![Screenshot 2022-05-01 at 21 10 28](https://user-images.githubusercontent.com/99187211/166160883-1ec39017-28d4-489f-80db-89f4a07c0fbd.png)

**Comment by someengineer-nordsec** [comment]
Maybe this will fix it https://github.com/MonitorControl/MonitorControl/pull/954

**Comment by the0neyouseek** [comment, marked as answer]
Hi @olso-nordsec 
Sadly, the hdmi port on the m1 macbook pro does not support DDC control, please see [this discussion #750](https://github.com/MonitorControl/MonitorControl/discussions/750)
